backend/src/config/archives.json

```diff
@@ -1,6 +1 @@
-[
-    {
-        "url": "wss://kusama-rpc.polkadot.io",
-        "path": "/Users/apple/Documents/kusama-archive-2021-oct-23"
-    }
-]
+[]
```

Comment:
if I remove it there will be an error `Cannot find module './config/archives.json'`, because we're importing this file by default, so I would keep it with empty array and just ignore updates